### PR TITLE
Feat/erlang hosts

### PR DESCRIPTION
**Pull request by Gazler**
Very similar to the functionality provided in https://erlang.org/doc/man/net_adm.html but with the logging benefits of libcluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +85,32 @@ currently under development. By default, the Erlang distribution is used.
 
 ### Clustering Strategies
 
+The `ErlanngHosts` strategy relies on having a `.erlang.hosts` file in one of the following locations as specified in
```
Comment (bitwalker) [inline README.md]: I think you have a typo here ;) (ErlanngHosts)

**Comment by bitwalker** [comment]
Looks good! Thanks for this, just one small tweak and I'll get this merged :)

**Comment by Gazler** [comment]
@bitwalker Done!

**Comment by bitwalker** [comment]
Thanks!

**Pull request merged**